presto-hive-metastore/src/main/java/com/facebook/presto/hive/metastore/Partition.java

```diff
@@ -127,6 +130,17 @@ public boolean isSealedPartition()
         return sealedPartition;
     }
 
+    @JsonProperty
+    public int getCreateTime()
+    {
+        return createTime;
+    }
+
+    public void setCreateTime(int createTime)
+    {
+        this.createTime = createTime;
+    }
```

Comment:
Lets remove this and use the partition.build() instead. see other comment.

Reply:
done.